app/src/main/kotlin/com/jakewharton/sa4p/ui/footer.kt

```diff
@@ -35,7 +35,10 @@ import com.jakewharton.sa4p.presenter.MainModel
 import com.jakewharton.sa4p.presenter.Unauthenticated
 
 @Composable
-fun BottomBar(model: MainModel) {
+fun BottomBar(
+	model: MainModel,
+	modifier: Modifier = Modifier,
```

Comment:
Same as above. Needs to be used by the root layout.

Reply:
BottomAppBar has its own modifier.